tsconfig.base.json

```diff
@@ -55,6 +56,7 @@
       "@keira/shared/switch-language": ["libs/shared/switch-language/src/index.ts"],
       "@keira/shared/test-utils": ["libs/shared/test-utils/src/index.ts"],
       "@keira/shared/utils": ["libs/shared/utils/src/index.ts"],
+      "keira-features-unused-guid-search": ["libs/features/unused-guid-search/src/index.ts"],
       "texts": ["libs/features/texts/src/index.ts"]
```

Comment:
we should make it consistent and use `@keira/features/...` like in the other places